tests/network-setups/fabric/dev/scripts/setupCCLocal.sh

```diff
@@ -5,34 +5,11 @@ INTEROP_CC_PATH=$PWD/../../../../core/network/fabric-interop-cc
 
 echo "Setting up Interop CC..."
 
-cd $CHAINCODE_PATH
-if [ -d "fabric-interop-cc" ]; then
-    echo "Deleting INTEROP CC in the test folder to copy the latest from the main folder...."
-    rm -rf fabric-interop-cc
-fi
-if [ -d "interop" ]; then
+if [ -d "${CHAINCODE_PATH}/interop" ]; then
     echo "Deleting previously built interop cc folder"
-    rm -rf interop
+    rm -rf ${CHAINCODE_PATH}/interop
 fi
-cp -r ${INTEROP_CC_PATH} .
-
-cd fabric-interop-cc
-cp -r $PWD/../../../../../../common/protos-go contracts/interop/
-cp -r $PWD/../../../../../../core/network/fabric-interop-cc/libs contracts/interop/
-cd contracts/interop/libs/assetexchange
-mv go.mod.local go.mod
-rm go.sum
-go mod tidy
-cd ../utils
-mv go.mod.local go.mod
-rm go.sum
-go mod tidy
-cd ../..
-mv go.mod.local go.mod
-rm go.sum
-go mod tidy
-cd ../..
-cp -R contracts/interop ../interop
-
+(cd $INTEROP_CC_PATH/contracts/interop && make run-vendor)
```

Comment:
Why do you need the parantheses?

Reply:
parantheses makes sure when the command ends, we are back to original directory, and not to the one where did cd to.